docs/content/usage-reporting.md

```diff
@@ -0,0 +1,217 @@
+
+NGINX Cluster Connector is a Kubernetes controller that connects to the NGINX Management Suite and reports the number of NGINX Ingress Controller nodes in the cluster. The NGINX Cluster Connector is deployed as a Kubernetes Deployment in the same cluster where NGINX Ingress Controller is deployed.
+
+To use the NGINX Cluster Connector, you must have access to NGINX Management Suite. For more information, see [NGINX Management Suite](https://www.nginx.com/products/nginx-management-suite/).
+
+## Requirements
+
+To deploy Usage Reporting, you must have the following:
+
+| Requirements                            | Notes                                                                                                                 |
+|-----------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| NGINX Ingress Controller 3.1.0 or later | https://docs.nginx.com/nginx-ingress-controller                                                                       |
+| NGINX Management Suite 2.11 or later    | https://docs.nginx.com/nginx-management-suite                                                                         |
+| Docker 23 or later                      | https://docs.docker.com/get-docker/                                                                                   |
+| Kubernetes 1.22 or later                | Ensure you can access the Kubernetes API, with the ability to deploy a Kubernetes Deployment and a Kubernetes secret. |
+| kubectl 1.22 or later                   | https://kubernetes.io/docs/tasks/tools/#kubectl                                                                       |
+
+In addition to the software requirements, you must have the following:
+- Access to NGINX Management Suite with username and password for basic authentication, including the URL, username, and password, with the access to the `/api/platform/v1/k8s-usage` endpoints
+- Access to the Kubernetes cluster where the NGINX Ingress Controller is deployed, with the ability to deploy a Kubernetes Deployment and a Kubernetes Secret.
+- Access to public internet to pull the image, since the image is hosted in the NGINX container registry at `registry.nginx.com/cluster-connector`. Alternatively, you can pull the image down and push it to a private container registry that your cluster has access to.
+[//]: # (  TODO: Update the image and tag after published)
+
+## Setting up user accounts in NGINX Management Suite
+
+The NGINX Cluster Connector uses the NGINX Management Suite API to report the number of NGINX Ingress Controller nodes in the cluster. To use the NGINX Management Suite API, you must have a user account with the correct role.
```

Comment:
The cluster connector needs a user account to send usage data to NIM.  This is how you create one for the Cluster Connector to use.